core/modules/query/QueryTemplate.h

```diff
@@ -201,6 +207,7 @@ class QueryTemplate {
 private:
     EntryPtrVector _entries;
     SetAliasMode _aliasMode{USE_ALIAS};
+    bool _quoteIdentifiers{true}; // if true, identifiers will be quoted.
```

Comment:
This isn't a request, just a suggestion to use the following form of the in-header initialization:
```
bool _quoteIdentifiers = true;    ///< if true, the identifiers will be quoted.
```